lib/companies_house/client.rb

```diff
@@ -38,8 +38,9 @@ def officers(id)
 
       loop do
         page = request(id, '/officers', start_index: offset)
-        total = page['total_results']
         new_items = page['items']
+        total = page['total_results'] || new_items.count
```

Comment:
it's worth inquiring into whether or not a response with nil `total_results` can be trusted to have all the results on the first page